ui/src/alerting/components/notifications/StatusChangeDropdown.tsx

```diff
@@ -0,0 +1,60 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+
+// Types
+import {StatusRuleItem} from 'src/types'
+
+// Components
+import {Dropdown} from '@influxdata/clockface'
+import {NewRuleDispatch} from 'src/alerting/components/notifications/NewRuleOverlay'
+
+// Utils
+import {changes, changeStatusRule, activeChange} from './statusChange'
```

Comment:
Can you put this in `src/alerting/utils`? I know you're not a fan of utils folders, but I think adding yet another file structure convention that isn't followed consistently is worse.

Reply:
what about StatusChange.utils.ts 😄